### ISSUE TITLE
nx migrate latest is not updating angular from 15 to 16 even when to option is provided

### ISSUE THREAD

**Issue by abdulkareemnalband** [nrwl/nx#17263]
### Current Behavior

on executing  `npx nx migrate latest   --to="@angular/core@latest,@angular/cli@latest,@angular/cdk@latest,@ngrx/store@latest"` no migrations are generated

### Expected Behavior

on executing  `npx nx migrate latest   --to="@angular/core@latest,@angular/cli@latest,@angular/cdk@latest,@ngrx/store@latest"` angular should be updated to version 16

### Steps to Reproduce

1. run  `npx nx migrate latest   --to="@angular/core@latest,@angular/cli@latest,@angular/cdk@latest,@ngrx/store@latest"`
2. Nothing happens

### Nx Report

```shell
Node   : 16.19.0
   OS     : win32 x64
   npm    : 8.19.3
   Hasher : Native

   nx                 : 16.2.2
   @nx/js             : 16.2.2
   @nx/jest           : 16.2.2
   @nx/linter         : 16.2.2
   @nx/workspace      : 16.2.2
   @nx/angular        : 16.2.2
   @nx/cypress        : 16.2.2
   @nx/devkit         : 16.2.2
   @nx/eslint-plugin  : 16.2.2
   @nrwl/tao          : 16.2.2
   @nx/webpack        : 16.2.2
   typescript         : 5.0.4
   ---------------------------------------
   Community plugins:
   @ngrx/component-store : 15.3.0
   @ngrx/effects         : 15.3.0
   @ngrx/entity          : 15.3.0
   @ngrx/router-store    : 15.3.0
   @ngrx/schematics      : 15.3.0
   @ngrx/store           : 15.3.0
   @ngrx/store-devtools  : 15.3.0
```

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by gnuechtel** [comment]
Try `npx nx@latest migrate latest`

**Comment by abdulkareemnalband** [comment]
This is fixed in 16.3 now
Previously `npx nx migrate latest` migrated nx libraries only
